src/fides/api/ops/task/graph_task.py

```diff
@@ -114,6 +116,20 @@ def result(*args: Any, **kwargs: Any) -> Any:
                     )
                     self.log_skipped(action_type, exc)
                     return default_return
+                except (SkippingConsentPropagation,) as exc:
+                    logger.warning(
+                        "Skipping collection {} for privacy_request: {}",
```

Comment:
nit: might it make sense to add something in the log message here about consent specifically, just to make it clearer which code path was hit at this point, for anyone going back through the logs? now that we've separated out this exception handler from the other more generic block above.

Reply:
good idea